layouts/channel.vue

```diff
@@ -0,0 +1,8 @@
+<template>
+  <div class="basis-64 bg-gray-800 p-3">
+    <slot name="channel" />
+  </div>
+  <main class="relative flex-1 bg-gray-700 p-3">
+    <slot name="chat" />
+  </main>
```

Comment:
plz wrap it in div and re-enable that eslint rule
i dont know why its a eslint restirction, but maybe it was done for a good reason. lets trust official vue plugin here :)